gnc/sub8_system_id/ReadMe.md

```diff
@@ -0,0 +1,21 @@
+Estimate Drag Coefficients
+==========================
+
+This program calculates the linear drag coeffiecnt in all six directions that the sub can move.
+
+# Usage
+
+run
+
+	roslaunch sub8_launch dynamics_simulator.launch
+
+In a new window, run
+
+	 rosnode kill /rise_6dof
+to kill the controller.
+
+run
+
+	rosrun sub8_system_id  estimate_drag_coefficients.py
```

Comment:
Command does not work as the file is not executable. You must run
`chmod +x <filename>`